src/api/sushi.ts

```diff
@@ -304,10 +315,18 @@ export async function getSushiPools(chain: Chain) {
           BigNumber.from(pair.reserve0),
           BigNumber.from(pair.reserve1),
         ],
-        vAPY: ftoBN(pair.daySnapshots[0].feesUSD),
-        tAPY: ftoBN("12"),
-        volume: ftoBN(pair.daySnapshots[0].volumeUSD),
-        tvl: ftoBN(pair.liquidityUSD),
+        apr: get24hFees(pair.hourSnapshots)
+          .mul(365)
+          .mul(100)
+          .mul(BigNumber.from(10).pow(PCT_DECIMALS))
+          .div(usdStrToBN(pair.liquidityUSD)),
+        volume: get24hVolume(pair.hourSnapshots).mul(
+          BigNumber.from(10).pow(USD_DECIMALS)
```

Comment:
you can move the 10^ decimals multiply into usdStrToBN no?

Reply:
I was using usdStrToBN for swapFee too
which is wrong
also swapFee was represented as 2 decimals and had to be sent as PCT_DECIMALS (which is also 2)
usd were represented as 0 decimals and had to be sent as USD_DECIMALS (which is also 2) 
so things were confusing
Hence this: https://github.com/jezer0x/barren-wuffet-ui2/pull/17/commits/2ca3164ed698d854d16240abe858f7125aa4a4e0